Containers/apache/Caddyfile

```diff
@@ -28,16 +28,19 @@
     route /browser/* {
         reverse_proxy {$COLLABORA_HOST}:9980 {
             # trusted_proxies placeholder
+            header_up Host {http.request.host}
```

Comment:
Not sure if this is really needed aa it doesnt make it work anyway. Question would then also be why it currently works even without it